packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/customTemplates.test.ts

```diff
@@ -38,40 +39,39 @@ describe('Custom Templates Create', () => {
   let openTextDocumentStub: SinonStub;
   let sendCommandEventStub: SinonStub;
   let sendExceptionStub: SinonStub;
-  let getConfigValue: SinonStub;
+  let getTemplatesDirectoryStub: SinonStub;
 
   beforeEach(() => {
-    showInputBoxStub = stub(vscode.window, 'showInputBox');
-    quickPickStub = stub(vscode.window, 'showQuickPick');
-    appendLineStub = stub(channelService, 'appendLine');
-    showSuccessfulExecutionStub = stub(
+    sandbox = createSandbox();
+    showInputBoxStub = sandbox.stub(vscode.window, 'showInputBox');
+    quickPickStub = sandbox.stub(vscode.window, 'showQuickPick');
+    appendLineStub = sandbox.stub(channelService, 'appendLine');
+    showSuccessfulExecutionStub = sandbox.stub(
       notificationService,
       'showSuccessfulExecution'
     );
     showSuccessfulExecutionStub.returns(Promise.resolve());
-    showFailedExecutionStub = stub(notificationService, 'showFailedExecution');
-    openTextDocumentStub = stub(vscode.workspace, 'openTextDocument');
-    sendCommandEventStub = stub(telemetryService, 'sendCommandEvent');
-    sendExceptionStub = stub(telemetryService, 'sendException');
-    getConfigValue = stub(ConfigUtil, 'getConfigValue');
-    getConfigValue.returns(undefined);
+    showFailedExecutionStub = sandbox.stub(
+      notificationService,
+      'showFailedExecution'
+    );
+    openTextDocumentStub = sandbox.stub(vscode.workspace, 'openTextDocument');
+    sendCommandEventStub = sandbox.stub(telemetryService, 'sendCommandEvent');
+    sendExceptionStub = sandbox.stub(telemetryService, 'sendException');
+    getTemplatesDirectoryStub = sandbox.stub(
+      ConfigUtil,
+      'getTemplatesDirectory'
+    );
+    getTemplatesDirectoryStub.returns(undefined);
```

Comment:
nit: you can combine these thow lines like ```getTemplatesDirectoryStub = sandbox.stub(
      ConfigUtil,
      'getTemplatesDirectory'
    ).returns(undefined);
    ```

Reply:
looking at the tests all of them set a returns value on the stub so I don't think you need to include it here.